x/launch/keeper/change_param.go

```diff
@@ -0,0 +1,51 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/tendermint/spn/x/launch/types"
+)
+
+// SetChangeParam set a specific changeParam in the store from its index
+func (k Keeper) SetChangeParam(ctx sdk.Context, changeParam types.ChangeParam) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.ChangeParamKeyPrefix))
+	b := k.cdc.MustMarshal(&changeParam)
+	store.Set(types.ChangeParamPath(
+		changeParam.Module,
+		changeParam.Param,
+	), b)
+}
+
+// GetChangeParam returns a changeParam from its index
+func (k Keeper) GetChangeParam(
```

Comment:
`ChangeParam` should also be indexed by the launch ID to determine which chain get param changed